app/lib/numeric_hacks.rb

```diff
@@ -19,12 +21,14 @@ def percentage_change_from(other)
   #   5.percentage_ratio_of(10)   # 50.0
   #   0.1.percentage_ratio_of(10) # 1.0
   def percentage_ratio_of(other)
-    if self.zero? && other.zero?
+    is_other_zero = other.zero?
+
+    if zero? && is_other_zero
       0.0
-    elsif other.zero?
+    elsif is_other_zero
       100.0
     else
-      (self.to_f / other) * 100.0
+      (to_f / other) * 100.0
     end
   end
```

Comment:
```ruby
def percentage_ratio_of(other)
  if zero?
    0.0
  elsif other.zero?
    100.0
  else
    (to_f / other) * 100.0
  end
end
```

Because `(0/X) * 100.0` is always 0 anyway, so the comparison of the 2 values being 0 to return 0 does not make sense to me and we only need to compare this one.